qp_klp/tests/test_step.py

```diff
@@ -146,6 +146,63 @@ def get(self, url):
         return None
 
 
+class AnotherFakeClient():
```

Comment:
Out of curiosity, how many fake clients? If many, can they be reused like:
```python
class FakeClientBase():
...
...

class FakeClient1(FakeClientBase)
...
...

class FakeClient2(FakeClientBase)
...
...
```

Reply:
There's just two. This new 'AnotherFakeClient()' just has a few specific samples to test how pre-check handles the situations we need it to. IMHO I don't think there's anything to be gained right now because most tests I think would use the first one, and they're really just wrappers to return dicts of metadata. If it became a reoccurring thing then perhaps, IMHO.